avm/res/machine-learning-services/workspace/main.bicep

```diff
@@ -515,6 +533,20 @@ type privateEndpointType = {
   resourceGroupName: string?
 }[]?
 
+type featureStoreSettingsType = {
```

Comment:
For consistency with others and as a not (yet) written rule, please keep the type naming convention in the form of <parameterNameInSingularForm>Type. So in this case, similarly to diagnosticSettingType below:
```suggestion
type featureStoreSettingType = {
```

Reply:
Updated